backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/IBPartnerDAO.java

```diff
@@ -316,6 +316,8 @@ I_C_BPartner_Location retrieveBillToLocation(Properties ctx,
 
 	BPartnerId cloneBPartnerRecord(@NonNull CloneBPartnerRequest request);
 
+	List<I_C_BPartner> retrieveVednors();
```

Comment:
```suggestion
	List<I_C_BPartner> retrieveVendors();
```
pls fix the typo :-)